### ISSUE TITLE
Cosmetic issue in font selection (dark mode)

### ISSUE THREAD

**Issue by tyrosinase** [inkstitch/inkstitch#3255]
![image](https://github.com/user-attachments/assets/cceb9429-84c3-4945-bbca-8bda154de10b)

If there's an easy way to give the picker an explicit white background, that should probably happen whenever someone happens to be updating the dialog. Even when you get some weirdo submitting a transparent-background preview.png it doesn't really help (kinda makes it worse actually; she should probably learn how to submit a patch with that fixed).

**Comment by tyrosinase** [comment]
... she just remembered that Bluenesia is not officially submitted at all, never mind that part

**Comment by kaalleen** [comment]
Hm, next up someone might create a white colored font and then complains about not seeing the letters because of the default white background ... I think it's best to leave it to the font authors to add a background color to the preview images...

**Comment by tyrosinase** [comment]
I was more thinking add a white background to the *dialog*, just to remove the dark bars from between the images. If someone made a white font and gave their particular preview a colored background, I don't think that would be a problem.

(in case it wasn't clear: the dingdong who always forgets to set a non-transparent background on the preview of their own dark font and then has their own system set to dark... is me. Different issue. 😆)

**Comment by kaalleen** [comment]
No worries, I saw the star beside of the font name ;)

I am still a bit opposed in doing it though. We then need to set font color and background color, then some systems don't want to accept color settings in dialogs and we got something invisible again. wxpython can be a bit picky about these things and if I don't have to, I rather don't mess with it. If somebody else wants to do the job, that's fine - when tested thoroughly on all operating systems with dark and light mode.
